packages/api/dev.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -eu
+
+# We assume we're running using `node` user
+[[ ! -d "node_modules" ]] && { echo "Installing modules..."; npm install; }
```

Comment:
Are you using a Linux machine? For those of us on MacOS it would be preferable to still be able to run locally with our system's `node_modules` intact.